click_repl/utils.py

```diff
@@ -159,8 +165,7 @@ def _help_internal():
             for description, mnemonics in info_table.items()
         )
 
-    val = formatter.getvalue()  # type: str
-    return val
+    print(formatter.getvalue())
```

Comment:
I think we should get rid of the print() and return it like before. what is your thoughts on this?